distribution/kernel/carbon-home/repository/resources/conf/templates/repository/conf/user-mgt.xml.j2

```diff
@@ -23,9 +23,9 @@
                 <UserName>{{super_admin.username}}</UserName>
                 <Password>{{super_admin.password}}</Password>
             </AdminUser>
-            <EveryOneRoleName>everyone</EveryOneRoleName>
+            <EveryOneRoleName>{{realm.every_one_role_name}}</EveryOneRoleName>
```

Comment:
Are the users changing this config?

Reply:
It is mentioned in the docs, can we omit?
https://docs.wso2.com/display/IS580/Configuring+the+System+Administrator

Reply:
Yes, let's remove this from config. If there's a requirement to have that for some reason, we can provide that later

Reply:
The template for this config was removed